stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -715,11 +716,56 @@ public class ChatClient internal constructor(
         return api.addDevice(device)
     }
 
+    @Deprecated(
+        message = "Use the searchMessages method with unwrapped parameters instead",
+        replaceWith = ReplaceWith("searchMessages(channelFilter, messageFilter, offset, limit)"),
+        level = DeprecationLevel.WARNING,
+    )
     @CheckResult
     public fun searchMessages(request: SearchMessagesRequest): Call<List<Message>> {
         return api.searchMessages(request)
     }
 
+    /**
+     * Search messages across channels. There are two ways to paginate through search results:
+     *
+     * * Using [limit] and [offset] parameters
+     * * Using [limit] and [next] parameters
+     *
+     * Pages of sort results will be returned with **next** and **previous** strings which
+     * can be supplied as a next parameter when making a query to get a new page of results.
```

Comment:
It would be nice to mention when we can use `sort` parameter

Reply:
Nice catch!